packages/admin/README.md

```diff
@@ -41,12 +79,13 @@ npx @api3/airnode-admin create-requester \
 ### `set-requester-admin`
 
 Sets the [requester admin](https://github.com/api3dao/api3-docs/blob/master/request-response-protocol/requester.md#requesteradmin).
-The `mnemonic` you provide here has to belong to the previous requester admin.
+The account derived from the `mnemonic` you provide here has to belong to the previous requester admin.
 
 ```sh
 npx @api3/airnode-admin set-requester-admin \
   --providerUrl https://ropsten.infura.io/v3/<KEY> \
   --mnemonic "nature about salad..." \
+  --derivationPath "m/44'/60'/0'/0/2" \
```

Comment:
I'm worried `derivationPath` is going to be more confusing than the benefit it provides

Reply:
We discussed this with Burak in https://api3workspace.slack.com/archives/C01N6001YCE/p1617872389030500

Basically, mnemonic alone is not enough. User might want to use specific account (derived from his mnemonic). I think advanced users should have an option to do so if they want.

Reply:
Ok thanks. As long as we can make it clear it's not required in most cases (and I see it's mentioned above)